.github/workflows/release.yml

```
@@ -6,17 +6,15 @@ on:
     tags:
       - "v*.*.*"
 jobs:
+  buildandtest:
+    uses: ./.github/workflows/go.yml
+    secrets: inherit
+
   build:
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-22.04"
```

We are running this on purpose on ubuntu 20.04 so we get older GLIBC compatability.

```suggestion
    runs-on: "ubuntu-20.04"
```

---

urgs. yeah, I vaguely remember 😞